megamek/src/megamek/codeUtilities/MathUtility.java

```diff
@@ -33,7 +33,7 @@ private MathUtility() {
      *         maximum. Otherwise, this will return the rounded integer between the
      *         two points
      */
-    public static int lerp(final int min, final int max, final double f) {
+    public static int lerp(int min, int max, double f) {
```

Comment:
`final` is useless in arguments, so I just removed it.

Reply:
It's actually not useless. It prevents the programmer from re-assigning a value to them.